.github/workflows/km-ci-workflow.yaml

```diff
@@ -136,17 +136,6 @@ jobs:
           mkdir /opt/kontain/bin/shim
           cp -r build/cloud/k8s/deploy/shim/* /opt/kontain/bin/shim
 
-      - name: internal release package
-        run: |
-          sudo apt-get update
-          sudo apt-get install makeself
-          make kkm-pkg
-          make release
-          mkdir /tmp/release
-          cp -r build/kontain.tar.gz /tmp/release
-          mkdir /tmp/bin_release
-          cp -r build/kontain_bin.tar.gz /tmp/bin_release
-
       - uses: actions/upload-artifact@v2
         with:
           name: km
```

Comment:
Three lines below - where is this krun coming from and what's the point?

Reply:
That's the dynamically linked krun that we were building before and we are still using for testing in the km-build step. I haven't transitioned that. If you think we need to we could open an issue.